scipy/spatial/transform/tests/test_rotation.py

```diff
@@ -621,6 +621,31 @@ def test_inv_single_rotation():
     assert_array_almost_equal(result2, eye3d)
 
 
+def test_identity_magnitude():
+    n = 10
+    assert (Rotation.identity(n).magnitude() == 0).all()
```

Comment:
Overall for floating point numbers a more adequate approach is to use `assert_allclose` from numpy.testing.